docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -487,7 +487,7 @@ sophisticated, `BSON <http://bsonspec.org/>`_.
 Implementing Game Play
 ======================
 
-Next, describe how to play tic-tac-toe. The details here are fairly
+This tutorial does not include example code for playing tic-tac-toe. The details are fairly
```

Comment:
I don't think the added text is relevant/accurate. In the previous section there are the CLI commands for playing the game. I'd delete this text up through "straightforward, and". Just begin with "The functionality could certainly ..."

Reply:
This section will be rewritten soon.  In the meantime, I'll delete the first few sentences and start with "The functionality could certainly...".